src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/ControllerUtil.java

```diff
@@ -0,0 +1,21 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca;
+
+/**
+ * ControllerUtil is a singleton which provides visibility into {@link RcaController}'s state
+ */
+public class ControllerUtil {
+    public static final ControllerUtil API = new ControllerUtil();
```

Comment:
I would suggest we don't use static classes or initializes as much as possible. 
The thing is your UT framework initializes once. If you set something in a static object in one of the tests, it stays there (unless explicitly reset after the tests pass). This can cause weird behaviors in your tests based on the order they execute etc.
If we can remove them, it will be great.

Reply:
I also want to avoid static classes / members where possible. That was the intent here. rcaEnabled previously was a static member of the RcaController that could only be set via a file read through a controller method. This causes issues for the reason you mentioned in your comment: testing with static classes/methods/members sucks. That's why I extracted the read/write logic for isRcaEnabled into the singleton class here. I did this in a few other places so that I could avoid static classes and bytecode hijacking via PowerMock.

Reply:
I removed the ControllerUtil, the RcaController can be refactored at a future time